hack/generator/Makefile

```diff
@@ -17,6 +17,10 @@ all: build test
 manifests: $(CONTROLLER_GEN)
 	$(Q) $(CONTROLLER_GEN) $(CRD_OPTIONS) rbac:roleName=manager-role webhook paths="./apis/..." output:crd:artifacts:config=config/crd/bases
 
+.PHONY: gen-arm
```

Comment:
I tried running `make gen-arm` with the output from a previous run still in `apis/` and I got lots of errors (compile errors in the generated code) - is it worth having a `make clean` or having this get rid of the previous output before building?

Reply:
I merged your PR which fixes this.